### ISSUE TITLE
Latency scale of ViT B16 on A100, FP32

### ISSUE THREAD

**Issue by kaimingkuang** [NVIDIA/FasterTransformer#626]
Hi,

I am trying to measure the latency of ViT implementations from different sources (CLIP, torchvision, etc). I run the model inference on A100 GPU with FP32 mode and PyTorch 1.13.0. When I use batch size of 32 and image size of 384x384, the latency roughly matches what you get (290.7 ms and 242 ms). Is this the latency of the entire batch or per image? I was told that the latency of ViT is normally somewhere around 30 ms per image, which is quite different from what I got. How do you do latency analysis? Here is my code:
```python
timings = np.zeros(50)
starter, ender = torch.cuda.Event(enable_timing=True), torch.cuda.Event(enable_timing=True)
with torch.no_grad():
    for i in tqdm(range(50)):
        images = torch.rand(32, 3, 384, 384).cuda()
        starter.record()
        outputs = pcv2_vit(images)
        ender.record()
        torch.cuda.synchronize()
        timings[i] = starter.elapsed_time(ender)
        print(timings[i])

print(timings.mean(), timings.std())
```
The bottom line is that what I get and [what you have here](https://github.com/NVIDIA/FasterTransformer/blob/main/docs/vit_guide.md#fp32-1) match, but both are a lot faster than what I thought. Is there anything I did wrong when testing the latency? Thank you very much!
